lib/suspenders/generators.rb

```diff
@@ -0,0 +1,30 @@
+require "active_support/concern"
+
+module Suspenders
+  module Generators
+    module APIAppUnsupported
+      class Error < StandardError
+        def message
+          "This generator cannot be used on API only applications."
+        end
+      end
+
+      extend ActiveSupport::Concern
+
+      included do
+        def raise_if_api_only_app
+          if api_only_app?
+            raise Suspenders::Generators::APIAppUnsupported::Error
+          end
+        end
+      end
```

Comment:
This will run first when we call `include Suspenders::Generators::APIAppUnsupported` in a generator. This is important because we don't want to invoke any of the generator's methods.